src/components/ErrorBoundary/ErrorBoundary.test.js

```diff
@@ -1,16 +1,14 @@
 import React from "react";
-import ErrorBoundary from "./ErrorBoundary";
-
 import { shallow } from "enzyme";
 
+import ErrorBoundary from "./ErrorBoundary";
+
 describe("Error Boundary", () => {
-  it("generates a error message when an error is caught", () => {
+  it("renders without crashing and matches snapshot", () => {
```

Comment:
< pedantic >You don't typically include how you're testing in the title. Just the what< /pedantic > ;-)